### ISSUE TITLE
404

### ISSUE THREAD

**Issue by refkoer** [refkoer/coming-soon-page#1]
@thomasjbradley

**Comment by thomasjbradley** [comment]
What's up?

**Comment by refkoer** [comment]
@thomasjbradley Yeah, it's the 404 issue page that I have been struggling with, it seems to be not showing up on the link, but I'm pretty sure that I linked everything correctly. 

It says that it cannot detect my index.html page

**Comment by thomasjbradley** [comment]
The `gh-pages` branch is spelled wrong, the “s” is missing.
